src/components/access_group/AccessGroupLeaveConfirmationNotification.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <article
+    class="notification-container leave-confirmation-notification-container"
+    v-if="showLeaveConfirmationNotification"
+  >
+    <span class="leave-confirmation-notification__icon">
+      <Icon id="question-mark-circle" :width="32" :height="32" />
+    </span>
+    <p
+      class="leave-confirmation-notification__description"
+      v-if="canLeaveGroup"
+    >
+      {{ confirmLeaveText }}
+    </p>
+    <p class="leave-confirmation-notification__description" v-else>
+      {{ leaveAlertText }}
+    </p>
+    <div class="leave-confirmation-notification__actions" v-if="canLeaveGroup">
+      <Button class="primary-action" v-on:click="handleLeaveClick()">{{
+        fluent('access-group_notifications', 'leave-action')
+      }}</Button>
+      <RouterLink
+        class="button secondary-action button--secondary button--action"
+        :to="{
+          name: 'Access Group',
+        }"
+        >{{ fluent('access-group_notifications', 'cancel-action') }}</RouterLink
+      >
+    </div>
+  </article>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import Icon from '@/components/ui/Icon.vue';
+import Button from '@/components/ui/Button.vue';
+import { ACCESS_GROUP_LEAVE_CONFIRMATION_PAGE } from '@/router';
+
+export default {
+  name: 'AccessGroupLeaveConfirmationNotification',
+  props: {},
+  components: { Icon, Button },
+  computed: {
+    ...mapGetters({
+      accessGroup: 'accessGroup/getGroup',
+      groupName: 'accessGroup/getGroupName',
+      getCurators: 'accessGroup/getCurators',
+      getProfile: 'userV2/getProfile',
+    }),
+    showLeaveConfirmationNotification() {
+      return this.$route.name === ACCESS_GROUP_LEAVE_CONFIRMATION_PAGE;
+    },
+    // Determine if the the user is allowed to leave the group, in case they are the last one in the group and the only curator
+    canLeaveGroup() {
```

Comment:
We should leave this to the API. Trying to leave as the last curator results in a 400 with 
```json
{"error":"Last admin of the group"}
```
If having this information before sending the request I can expose this. But in terms of concurrency I think just try and fail is fine. It's also something that rarely happens.